characterEntities/package-info.java

```diff
@@ -1,8 +0,0 @@
-/**
- * 
- */
-/**
- * @author Cathy
- *
- */
-package characterEntities;
```

Comment:
it was empty and useless, just add it back when there's something worth putting in there